Dockerfile

```diff
@@ -15,6 +15,8 @@ RUN yum update -y && yum install -y wget \
   && mvn package -Dmaven.test.skip=true \
   && mv target/perseo-core-*.war /usr/share/tomcat/webapps/perseo-core.war \
   && mkdir /var/log/perseo && chown tomcat:tomcat /var/log/perseo
+  # Force Tomcat to use ipv4 and optimize
+  && echo "JAVA_OPTS=\"-Djava.awt.headless=true -Xmx512m -XX:MaxPermSize=256m -XX:+UseConcMarkSweepGC -Djava.library.path=/usr/lib64:/usr/lib -Djava.security.egd=file:/dev/./urandom -Djava.net.preferIPv4Stack=true\"" >> /usr/share/tomcat/conf/tomcat.conf
```

Comment:
By now, we need here use ipv4, because perseo-fe try to connect with ipv4.
You can verify this inside perseo-fe?